server/@types/Assessment.ts

```diff
@@ -1,6 +1,18 @@
 export type AnswerDTOs = Record<string, AnswerDto>
 export type OasysEquivalent = Record<string, string | string[]>
 
+// Anything higher than this value is considered high scoring
+// the names of these fields match the CriminogenicNeedsData interface
+export const AssessmentAreaThreshold = new Map<string, number>([
+  ['accommodation', 1],
+  ['educationTrainingEmployability', 1],
+  ['drugMisuse', 0],
+  ['alcoholMisuse', 1],
+  ['personalRelationshipsAndCommunity', 1],
+  ['thinkingBehaviourAndAttitudes', 2],
+  ['lifestyleAndAssociates', 1],
+])
+
```

Comment:
This is probably not the right place to put this, @types is reserved for types/interfaces

Reply:
OK - any suggestion where? I don't see a natural home.

Reply:
The AssessmentService maybe?

Reply:
in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/240/commits/e97f4671278e119affe375dd6a9f7e0744760f37 i've moved this to its own file to avoid issues with mocking - it's in the /services/ dir now which I still think is the wrong place!